packages/flutter_tools/test/commands.shard/permeable/build_apk_test.dart

```diff
@@ -129,7 +129,7 @@ void main() {
 
     group('AndroidSdk', () {
       testUsingContext('throws throwsToolExit if AndroidSdk is null', () async {
-        final String projectPath = await createProject(tempDir, arguments: <String>['--no-pub', '--template=app']);
+        final String projectPath = await createProject(tempDir, arguments: <String>['--no-pub', '--template=app', '--platform=android']);
```

Comment:
why did you need to update these tests?

Reply:
So I didn't have to mock out the iOS signing mechanism in an Android-only test.

Reply:
:+1: